extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -344,10 +345,29 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 			await this.populateLocationDropdown();
 
 			// Collect source login info here, as it will speed up loading the next page
-			const sourceLogins: LoginTableInfo[] = [];
+			var sourceLogins: LoginTableInfo[] = [];
```

Comment:
why are we changing this from const to var?

Reply:
Previously we were not modifying its contents. But now we remove system logins from sourceLogins in a separate vairable.